digits/views.py

```diff
@@ -90,6 +90,11 @@ def home(tab=2):
                     'url': flask.url_for(
                         'digits.pretrained_model.views.new'),
                     },
+                'access-model-store': {
+                    'title': 'Retrieve from Model Store',
+                    'id': 'retrieveModelStore',
+                    'url': '/static/store.html',
```

Comment:
Thanks.  Yes, it will be much better to do flask.url_for on this.  I changed codes to reflect this.